setup_pip.py

```diff
@@ -109,12 +109,12 @@ def run(self):
 cuda_version = ''.join(os.environ.get('CUDA', 'unknown').split('.')[:2])
 
 install_requires = [
-    'pandas>=0.20,<0.21',
+    'pandas>=0.23.4',
     'numba>=0.40.0,<0.42',
     'pycparser==2.19',
-    'pyarrow>=0.10,<0.11',
+    'pyarrow==0.11.1',
     'cffi>=1.0.0',
-    'cython>=0.28,<0.29',
+    'cython>=0.29,<0.30',
```

Comment:
discrepancy between the conda env and the required cython here (i.e. `<0.30`) could lead us to fail at some point, no? Also I haven't seen packages build for 0.3, but I believe they are changing versioning to 3.0 for it.

Reply:
This ensures we get a 0.29.* build and prevents 0.30.* builds, isn't this what we want?

Reply:
yes, but it should coincide with the env (I guess I should have made the comment in the env instead where it is only `>0.29`)

Reply:
Ah okay, will update env accordingly.

Reply:
Fixed.